tests/kola-ci-self/tests/kola/exclusive-readonly

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# kola: { "isolation": "dynamicuser" }
```

Comment:
can you explain what this test is doing?

Reply:
Perhaps ironically...I do think the actual description of the test is something that we should represent in a structured way and indeed perhaps require.  

Anyways done